src/xmipp/applications/scripts/tomogram_reconstruction/tomogram_reconstruction.py

```diff
@@ -231,19 +252,52 @@ def run(self):
         
         self.readInputParams()
         
-        mdTs, ts = self.readTiltSeries(self.fnAngles)
-        
-        tiltAngles = []
-        for objId in mdTs:
-            tiltAngles.append(mdTs.getValue(xmippLib.MDL_ANGLE_TILT, objId))
-        
+        ts = self.readTiltSeries()
+        print(os.path.splitext(self.fnAngles)[1])
+        if os.path.splitext(self.fnAngles)[1] == '.tlt':
+            tiltAngles = self.readTltFile(self.fnAngles)
+        else:
+            mdTs = xmippLib.MetaData(os.path.expanduser(self.fnAngles))
+            tiltAngles = []
+            for objId in mdTs:
+                tiltAngles.append(mdTs.getValue(xmippLib.MDL_ANGLE_TILT, objId))
+            
         tiltAngles  = np.array(tiltAngles) * np.pi/180.0
 
         if self.useAstra:
             self.astraReconstruction(ts, tiltAngles, 'FP3D_CUDA')
         else:
             self.tigreReconstruction(ts, tiltAngles)
 
+    def fixMethod(self, method):
+        
+        method = method.replace("-","")
+        method = method.replace("_","")
+        
+        return method
+
+
+    def readTltFile(self, fnTlt):
+        try:
+            with open(fnTlt, 'r') as tlt:
+                lines = tlt.readlines()
+    
+                # Regular expression for matching numbers in the first column
+                number_pattern = re.compile(r'^\s*([\d.-]+)')
+    
+                # Try to extract the first column using a comma or whitespace as the delimiter
+                first_column = []
+                for line in lines:
+                    match = number_pattern.match(line)
+                    if match:
+                        first_column.append(float(match.group(1)))
+    
+                return np.array(first_column)
+    
+        except IOError:
+            raise FileNotFoundError("Error parsing the tlt file")
+    
+
 
     def astraReconstruction(self, ts_orig, tiltAngles, recAlgorithm):
         
```

Comment:
This pascalcase is against the naming convention in this file and also is considered a war crime

Reply:
Some IDEs might mistake this with a class symbol and show incorrect colours